matsim/src/main/java/org/matsim/core/mobsim/qsim/qnetsimengine/flow_efficiency/DefaultFlowEfficiencyCalculator.java

```diff
@@ -1,11 +1,13 @@
 package org.matsim.core.mobsim.qsim.qnetsimengine.flow_efficiency;
 
+import org.matsim.api.core.v01.Id;
 import org.matsim.api.core.v01.network.Link;
-import org.matsim.vehicles.Vehicle;
+import org.matsim.core.mobsim.qsim.qnetsimengine.QVehicle;
+import org.matsim.lanes.Lane;
 
 public class DefaultFlowEfficiencyCalculator implements FlowEfficiencyCalculator {
 	@Override
-	public double calculateFlowEfficiency(Vehicle vehicle, Link link) {
-		return vehicle.getType().getFlowEfficiencyFactor();
+    public double calculateFlowEfficiency(QVehicle qVehicle, QVehicle previousQVehicle, Double previousTime, Link link, Id<Lane> laneId) {
```

Comment:
previousTime should be previousTimeDiff  now

Reply:
Or maybe timeGapToPreviousVehicle? Or timeDiffToPreviousVehicle?

Reply:
I like `timeGapToPreviousVehicle`. But initially I just wanted to point out that the implemented class doesn't use the same name as in the interface. If you rename it, then it should be consistent, as others might use these as examples.